components/ad/extensionAdBlockDetector/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@schibstedspain/sui-ad-extensionAdBlockDetector",
```

Comment:
The name of the package is not correct. Use kebab-case instead camelCase.

Reply:
Fixed 👍